src/ExtendedGridOrbitals.h

```diff
@@ -96,8 +96,9 @@ class ExtendedGridOrbitals : public Orbitals
 
     void multiply_by_matrix(
         const DISTMATDTYPE* const, ORBDTYPE*, const int) const;
-    void multiply_by_matrix(const dist_matrix::DistMatrix<DISTMATDTYPE>& matrix,
-        ORBDTYPE* const product, const int ldp);
+    // void multiply_by_matrix(const dist_matrix::DistMatrix<DISTMATDTYPE>&
+    // matrix,
+    //    ORBDTYPE* const product, const int ldp);
```

Comment:
Why not removing the function instead of commenting it? The problem with commenting the code is that it makes it harder to find what you are looking for because there is "false positive". If you need this code later you can always look at the git history. It's really easy on github, you can just look at any file and look at the history/blame of the file.

Reply:
sure. I need to change a few other things in this branch anyway